tests/test_debian_clusters.py

```diff
@@ -37,13 +69,39 @@ def test_001_custom_single():
     # verify postgresql.conf is in data directory now
     assert os.path.exists(postgres_conf_path)
 
+    pgversion = os.getenv("PGVERSION")
+
+    p = subprocess.run(
+        [
+            "ls",
+            "-ld",
+            node1_path,
+            "/var/lib/postgresql/%s" % pgversion,
+            "/etc/postgresql/%s" % pgversion,
+            "/etc/postgresql/%s/debian_node1" % pgversion,
+            "/etc/postgresql/%s/debian_node1/postgresql.conf" % pgversion,
+            "/etc/postgresql/%s/debian_node1/pg_hba.conf" % pgversion,
+            "/etc/postgresql/%s/debian_node1/pg_ident.conf" % pgversion,
+        ],
+        text=True,
+        capture_output=True,
+    )
+    print("%s" % p.stdout)
+
     monitor.print_state()
 
-    node1.destroy()
 
-    # node is stuck in init state, destroy won't remove it from monitor
-    # force remove from monitor
-    monitor.run_sql_query(
-        "select pgautofailover.remove_node('%s', %s)"
-        % (str(node1.vnode.address), str(node1.port))
+def test_002_chmod_debian_data_directory():
```

Comment:
Why is this needed? No actual test is done after.

Reply:
It's needed so that we can destroy the nodes, otherwise we have a permission denied issue on removing PGDATA. So the tests would fail in the teardown method.